internal/config.go

```diff
@@ -15,52 +13,31 @@ type Config struct {
 	ChannelID string `yaml:"channel"`
 	LeadDevID string `yaml:"leadev"`
 	AdminID   string `yaml:"admin"`
+	Location  string `yaml:"location"`
 }
 
 // GetConfig retrieves a configuration.
 func GetConfig(configPath string) Config {
-	if _, err := os.Stat(configPath); err != nil {
-		genConfig(configPath)
-		panic("Please populate the new config file.")
-	}
-
-	file, err := ioutil.ReadFile(configPath)
-
-	if err != nil {
-		genConfig(configPath)
-		log.Fatalln("Failed to read configuration file. " + err.Error())
-	}
-
-	config := Config{}
-
-	if err = yaml.Unmarshal(file, &config); err != nil {
-		log.Fatalln("Failed to parse configuration file. " + err.Error())
-	}
-
-	return config
-}
-
-// Generate a configuration.
-func genConfig(configPath string) {
 	config := Config{
 		Token:     "",
 		Prefix:    ".mention",
```

Comment:
This could be made blank

Reply:
What do you mean?

Reply:
The string literal ".mention" could be empty or removed which will result in an empty string.

Reply:
Ah gotya